pages/EC-WebSphere_help.xml

```diff
@@ -3573,6 +3573,11 @@
         </div>
         
         <h1 id="rns">Release Notes</h1>
+        <h2>@PLUGIN_KEY@ 2.2.2</h2>
+        <ul>
+            <li>Added new procedures - SyncNodes, CheckNodeStatus</li>
+            <li>Made SyncCells and StartApplication configurable at DeployEnterpriseApplicationProcedure.</li>
```

Comment:
DeployEnterpriseApplication procedure now has configurable parameters - "Sync Cells" (sync cells after application deploy) and "Start Application" (starts the deployed application after the deploy process is finished)